src/openloco/windows/about_music.cpp

```diff
@@ -0,0 +1,187 @@
+#include "../graphics/colours.h"
+#include "../graphics/gfx.h"
+#include "../interop/interop.hpp"
+#include "../localisation/string_ids.h"
+#include "../objects/interface_skin_object.h"
+#include "../objects/objectmgr.h"
+#include "../windowmgr.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::about_music
+{
+    constexpr uint16_t ww = 500;
+    constexpr uint16_t wh = 312;
+
+    namespace widx
+    {
+        enum
+        {
+            frame,
+            title,
+            close,
+            panel,
+            scrollview,
+        };
+    }
+
+    static widget_t _widgets[] = {
+        make_widget({ 0, 0 }, { ww, wh }, widget_type::frame, 0),
+        make_widget({ 1, 1 }, { ww - 2, 13 }, widget_type::caption_25, 0, string_ids::music_acknowledgements_caption),
+        make_widget({ ww - 15, 2 }, { 13, 13 }, widget_type::wt_9, 0, 2321, string_ids::tooltip_close_window),
+        make_widget({ 0, 15 }, { ww, 297 }, widget_type::panel, 1),
+        make_widget({ 4, 18 }, { ww - 8, 289 }, widget_type::scrollview, 1, 2),
+        widget_end(),
+    };
+
+    static window_event_list _events;
+
+    static void on_mouse_up(ui::window* window, widget_index widgetIndex);
+    static void get_scroll_size(ui::window* window, uint32_t scrollIndex, uint16_t* scrollWidth, uint16_t* scrollHeight);
+    static void tooltip(ui::window* window, widget_index widgetIndex);
+    static void draw(ui::window* window, gfx::drawpixelinfo_t* dpi);
+    static void draw_scroll(ui::window* window, gfx::drawpixelinfo_t* dpi, uint32_t scrollIndex);
+
+    // 0x0043B26C
```

Comment:
Incorrect; copy/paste error. Should probably just leave it out; the window is only opened through the about window's `on_mouse_up` event.

Reply:
You can use the address of `loc_43B4AF`

Reply:
OK. Would you like me to add addresses for the event subs as well?

Reply:
Yeah, just for completeness' sake